documentation/src/main/asciidoc/server_guide/management.adoc

```diff
@@ -76,15 +76,13 @@ In domain mode the port will be the port of the Remoting connector for the Serve
 
 ----
 
-//Community Only (JDG-3364)
-ifndef::productized[]
-=== Jolokia
+=== Exposing JMX Beans to Prometheus
 
-You can also expose JMX beans using link:https://jolokia.org/[Jolokia]. In order to do this, just run the server with
-additional parameter `--jmx`, for example: `./standalone.xml -c cloud.xml --jmx`. Jolokia configuration is stored in
-`agent.conf` file. It is possible to override this file by specifying it after `--jmx` parameter. For example:
-`./standalone.sh -c cloud.xml --jmx my-config.properties`.
-endif::productized[]
+* Start {brandname} servers as follows:
++
+----
+$ ./standalone.sh -c cloud.xml --jmx 8180:my-config.yaml
```

Comment:
Have we verified that this works ?

Reply:
I just tried with 9.4.17.Final and `./bin/standalone.sh --jmx 8180:bin/prometheus_config.yaml` works